src/transformers/models/plbart/tokenization_plbart_fast.py

```diff
@@ -0,0 +1,275 @@
+        "uclanlp/plbart-refine-java-medium": "https://huggingface.co/uclanlp/plbart-refine-java-medium/resolve/main/sentencepiece.bpe.model",
+        "uclanlp/plbart-refine-java-small": "https://huggingface.co/uclanlp/plbart-refine-java-small/resolve/main/sentencepiece.bpe.model",
+        "uclanlp/plbart-ruby-en_XX": "https://huggingface.co/uclanlp/plbart-ruby-en_XX/resolve/main/sentencepiece.bpe.model",
+    },
+    "tokenizer_file": {
+        "uclanlp/plbart-base": "https://huggingface.co/uclanlp/plbart-base/resolve/main/tokenizer.json",
+        "uclanlp/plbart-c-cpp-defect-detection": "https://huggingface.co/uclanlp/plbart-c-cpp-defect-detection/resolve/main/tokenizer.json",
+        "uclanlp/plbart-cs-java": "https://huggingface.co/uclanlp/plbart-cs-java/resolve/main/tokenizer.json",
+        "uclanlp/plbart-en_XX-java": "https://huggingface.co/uclanlp/plbart-en_XX-java/resolve/main/tokenizer.json",
+        "uclanlp/plbart-go-en_XX": "https://huggingface.co/uclanlp/plbart-go-en_XX/resolve/main/tokenizer.json",
+        "uclanlp/plbart-java-clone-detection": "https://huggingface.co/uclanlp/plbart-java-clone-detection/resolve/main/tokenizer.json",
+        "uclanlp/plbart-java-cs": "https://huggingface.co/uclanlp/plbart-java-cs/resolve/main/tokenizer.json",
+        "uclanlp/plbart-java-en_XX": "https://huggingface.co/uclanlp/plbart-java-en_XX/resolve/main/tokenizer.json",
+        "uclanlp/plbart-javascript-en_XX": "https://huggingface.co/uclanlp/plbart-javascript-en_XX/resolve/main/tokenizer.json",
+        "uclanlp/plbart-php-en_XX": "https://huggingface.co/uclanlp/plbart-php-en_XX/resolve/main/tokenizer.json",
+        "uclanlp/plbart-python-en_XX": "https://huggingface.co/uclanlp/plbart-python-en_XX/resolve/main/tokenizer.json",
+        "uclanlp/plbart-refine-java-medium": "https://huggingface.co/uclanlp/plbart-refine-java-medium/resolve/main/tokenizer.json",
+        "uclanlp/plbart-refine-java-small": "https://huggingface.co/uclanlp/plbart-refine-java-small/resolve/main/tokenizer.json",
+        "uclanlp/plbart-ruby-en_XX": "https://huggingface.co/uclanlp/plbart-ruby-en_XX/resolve/main/tokenizer.json",
+    },
+}
+
+PRETRAINED_POSITIONAL_EMBEDDINGS_SIZES = {
+    "uclanlp/plbart-base": 1024,
+    "uclanlp/plbart-c-cpp-defect-detection": 1024,
+    "uclanlp/plbart-cs-java": 1024,
+    "uclanlp/plbart-en_XX-java": 1024,
+    "uclanlp/plbart-go-en_XX": 1024,
+    "uclanlp/plbart-java-clone-detection": 1024,
+    "uclanlp/plbart-java-cs": 1024,
+    "uclanlp/plbart-java-en_XX": 1024,
+    "uclanlp/plbart-javascript-en_XX": 1024,
+    "uclanlp/plbart-php-en_XX": 1024,
+    "uclanlp/plbart-python-en_XX": 1024,
+    "uclanlp/plbart-refine-java-medium": 1024,
+    "uclanlp/plbart-refine-java-small": 1024,
+    "uclanlp/plbart-ruby-en_XX": 1024,
+}
+
+FAIRSEQ_LANGUAGE_CODES = [
+    "java",
+    "python",
+    "en_XX",
+]
+
+
+class PLBartTokenizerFast(XLMRobertaTokenizerFast):
```

Comment:
Let's not use inheritance here, should add this as a standalone tokenizer.

Reply:
@patil-suraj 
Removing fast tokenizer for now.